kpops/pipeline_generator/pipeline.py

```diff
@@ -104,27 +99,7 @@ def __add_ingoing_edges(
 
     @staticmethod
     def __get_vertex_component_name(component: PipelineComponent) -> str:
-        return f"component-{component.name}"
-
-    def __get_all_output_topics(self, component: PipelineComponent) -> list[str]:
-        all_output_topics: list[str] = []
-        output_topic = component.get_output_topic()
-        extra_output_topics = component.get_extra_output_topics()
-        if output_topic is not None:
-            all_output_topics.append(output_topic)
-        if extra_output_topics:
-            all_output_topics.extend(list(extra_output_topics.values()))
-        return all_output_topics
-
-    def __get_all_input_topics(self, component: PipelineComponent) -> list[str]:
-        input_topics = component.get_input_topics()
-        extra_input_topics = component.get_extra_input_topics()
-        all_input_topics: list[str] = []
-        if input_topics is not None:
-            all_input_topics.extend(input_topics)
-        if extra_input_topics:
-            all_input_topics.extend(chain(*extra_input_topics.values()))
-        return all_input_topics
+        return f"component-{component.full_name}"
```

Comment:
is the `component-` prefix really necessary?

Reply:
I mean... you can call the component as the same name of the topic